utils/stress-ng/patches/002-disable-compiler-test.patch

```diff
@@ -1,13 +0,0 @@
---- a/Makefile.config
```

Comment:
Would be good to mention in the commit description that this patch was intentionally removed.

Reply:
done

Reply:
Thank you! :)